src/hooks/use-sync-sites.ts

```diff
@@ -1,92 +1,126 @@
-import { useState, useEffect } from 'react';
+import * as Sentry from '@sentry/electron/renderer';
+import { useState, useEffect, useRef } from 'react';
+import { useAuth } from './use-auth';
+import { useOffline } from './use-offline';
 
-export type SyncSupport = 'unsupported' | 'syncable' | 'needs-transfer' | 'already-connected';
+const BUSINESS_PLAN_ID = 1008;
+const COMMERCE_PLAN_ID = 1011;
 
-export interface SyncSite {
+type SyncSupport = 'unsupported' | 'syncable' | 'needs-transfer' | 'already-connected';
+
+type SyncSite = {
```

Comment:
We need to export this type, so it can be used in different files.

```suggestion
export type SyncSite = {
```

Reply:
It's not used anywhere else yet, though, which is why I removed the export statement. Let's add it back when needed?

Reply:
It's used on https://github.com/Automattic/studio/blob/65568a1c95bbfe040a1375792f54ee6d9175c96a/src/components/sync-sites-modal-selector.tsx#L5

And that's why the CI tests are failing.

Reply:
Geez. My bad 🤦‍♂️

Reply:
I only caught it because the CI was failing 😆 .
BTW, Thanks for changing the interfaces to types 🙏